proto/ibc/applications/transfer/v1/transfer.proto

```diff
@@ -20,11 +20,13 @@ message Params {
 }
 
 // Forwarding defines a list of port ID, channel ID pairs determining the path
-// through which a packet must be forwarded, and the memo string to be used in the
-// final destination of the tokens.
+// through which a packet must be forwarded, and an unwind boolean indicating if
+// the coin should be unwinded to its native chain before forwarding.
 message Forwarding {
-  repeated Hop hops = 1 [(gogoproto.nullable) = false];
-  string       memo = 2;
+  // optional unwinding for the token transfered
+  bool unwind = 1;
+  // optional intermediate path through which packet will be forwarded
```

Comment:
Nit: intermediate sounds a little strange here, since the path includes the endpoint?